tests/AutoReview/CiConfigurationTest.php

```diff
@@ -74,12 +74,27 @@ public function testDeploymentJobsRunOnLatestStablePhpThatIsSupportedByTool()
         }
     }
 
-    private static function assertUpcomingPhpVersionIsCoveredByCiJob($lastSupportedVersion, array $ciVersions)
+    private static function ensureTraversableContainsIsAvailable()
```

Comment:
What is the purpose here?

It is named "ensure `TraversableContains` is available" and when it does not exist we mark it as skipped.

Reply:
Yup, that's what it does. Is there a problem?

Reply:
Contrary to what the diff shows, `assertUpcomingPhpVersionIsCoveredByCiJob` is still here.